sdks/java/core/src/main/java/org/apache/beam/sdk/testing/TestPipeline.java

```diff
@@ -155,6 +156,8 @@ public static PipelineOptions testingPipelineOptions() {
         options.as(GcpOptions.class).setGcpCredential(new TestCredential());
       }
       options.setStableUniqueNames(CheckEnabled.ERROR);
+
+      IOChannelUtils.registerStandardIOFactories(options);
```

Comment:
If you got time, you could make IOChannelUtils ServiceLoader aware and that would help a lot with https://issues.apache.org/jira/browse/BEAM-59 and would completely get rid of IOChannelUtils.registerStandardIOFactories